packages/go/openapi/src/paths/analysis.meta-trees.id.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+parameters:
+  - $ref: './../parameters/header.prefer.yaml'
+  - name: domain_id
+    description: Domain ID
+    in: path
+    required: true
+    schema:
+      type: string
+get:
+  operationId: GetComboTreeGraph
+  summary: Get the graph for meta tree
+  # TODO: Somebody who understands this endpoint better should improve this description

```

Comment:
I assume this is left from the previous large PR and not intended to be addressed as part of this one.

Looks good, just wanted to double check on the TODOs, though